modules/sharedaddy/sharing.js

```diff
@@ -37,13 +37,17 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 					]
 				};
 
-				for ( service in urls ) {
+				for ( service in requests ) {
 					if ( ! jQuery( 'a[data-shared=sharing-' + service + '-' + id  + ']' ).length ) {
 						continue;
 					}
 
-					while ( ( service_url = urls[ service ].pop() ) ) {
-						jQuery.getScript( service_url );
+					while ( ( service_request = requests[ service ].pop() ) ) {
+						if ( 'string' === typeof service_request ) {
```

Comment:
It's not hurting anything, but this condition ought to be removed, i.e. only use `jQuery.getScript` ([as noted in an earlier comment](https://github.com/Automattic/jetpack/pull/2816/files#r41426366)). If you'd prefer, we can wait until this is removed from the WPCOM side.